packages/forms-web-app/src/dynamic-forms/validator/conditional-required-validator.js

```diff
@@ -0,0 +1,53 @@
+const { body } = require('express-validator');
+const BaseValidator = require('./base-validator.js');
+
+/**
+ * @typedef {import('../question.js')} Question
+ */
+
+/**
+ * enforces a field is not empty when condition is satisfied
+ * @class
+ */
+class ConditionalRequiredValidator extends BaseValidator {
```

Comment:
looks good, can you put in some unit tests? see other validators for examples - need to test constructor and validate method